framework/src/main/java/org/checkerframework/common/reflection/DefaultReflectionResolver.java

```diff
@@ -610,6 +610,11 @@ private Symbol getSymbol(String className, Env<AttrContext> env, Names names, Re
         }
     }
 
+    @SuppressWarnings("ASTHelpersSuggestions") // Use local helper.
```

Comment:
```suggestion
    /**
     * Determine the enclosed elements for an element.
     * This wrapper is useful to avoid a signature change in the called method.
     *
     * @param sym the element
     * @return the enclosed elements
     */
    @SuppressWarnings("ASTHelpersSuggestions") // Use local helper.
```